config/admin.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+return [
+    'sidebar' => [
+        'dashboard' => [
+            'title' => 'dashboard',
+            'route' => 'admin.dashboard',
+            'icon' => '<i class="bi bi-grid"></i>'
```

Comment:
```suggestion
            'icon' => '<i class="bi bi-grid"></i>',
```
bổ sung dấu phẩy ở phần tử cuối